README.md

```diff
@@ -1,43 +1,33 @@
-##Short description of the Ipython 3 to/from Python converter files.
+### Short description of the Jupyter notebook to/from Python converter files.
 
-The *ipython notebook --script* run option that produced python codes
-from an Ipython 2 notebook is deprecated in Ipython 3 (Jupyter, nbformat 4).
-The python file that Jupyter can produce has several shortcomings, the most
-problematic one is the missing metainformation: there is no cell type
-metadata, just a bracket indicates where a new code cell starts, but information
-about markdowncells are completly lost. This is why restoring the
-notebook (without the output content) without losing important
-information is not possible right now.
+It can be difficult to version control Jupyter notebooks, especially if you don't want to track changes in the output, just the code.  While Jupyter can generate python files from a notebook, there is no cell type metadata and information about markdown cells is completely lost.
 
+#### To automatically save your Jupyter notebook as a Python file with metadata, add the text in **jupyter_notebook_config.txt** to your jupyter_notebook_config.py file.
 
-The scripts in the folder suggest a solution to this problem.
+* On Windows, this file can generally be found in the C:\Users\YOUR_USER_NAME\\.jupyter directory.  If there is no config file in this directory, create one with ```jupyter notebook --generate-config``` at the command line.
```

Comment:
I would add the lines about Windows instructions as a remark or so and would not put it in the first place. The authors of the scripts uses Linux and it is implicitly assumed that the users also use some Linux/Unix distribution. Once I have tried on Windows and for some reason it did not work (different end-of-line character or something with encoding that caused the error). It might work on it but I have never checked this properly.